stories/PInput.stories.js

```diff
@@ -0,0 +1,37 @@
+import { action } from '@storybook/addon-actions';
+import PInput from '../src/components/PInput.vue';
+import Render from './Render';
+
+export default {
+  title: 'PInput'
+};
+
+export const basicUsage = () => Render({
+  components: { PInput },
+  template: '<PInput @focus="action" name="input_basic" v-model="value" @input="myLog" />',
```

Comment:
Don't forget to create a `data` and have `value` there as an empty string on all examples.

Would be great if `myLog` logged `value`.

Reply:
I really can't get `value` logged... Any help?

Reply:
Something like this:

```
export const basicUsage = () => Render({
  components: { PInput },
  template: '<PInput @focus="action" name="input_basic" v-model="name" @input="myLog(name)" />',
  data: () =>({ name: '' }),
  methods: { action: action('focused'), myLog: action('input') }
});
```